src/trivia/libraries/trivia/trivia.go

```diff
@@ -0,0 +1,91 @@
+package trivia
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+	"trivia/libraries/prize"
+	"trivia/libraries/problems"
+)
+
+const limitSeconds = 60
```

Comment:
I think you're actually taking this in as a flag and don't use this number

Reply:
oh this is the default, in which case "limit" is what should go in the header print

Reply:
I am not sure which "limit" you mean